pipeline_dp/dp_engine.py

```diff
@@ -42,3 +42,31 @@ def aggregate(self, col, params: AggregateParams,
     # TODO: implement aggregate().
     # It returns input for now, just to ensure that the an example works.
     return col
+
+  def _bound_cross_partition_contributions(self, col, max_partitions_contributed: int, max_contributions_per_partition: int, aggregator_fn):
+    """
+    Binds the contribution by privacy_id within and across partitions
+    Args:
+      col:collection, with types of each element: (privacy_id, partition_key, value)
+      max_partitions_contributed
```

Comment:
Please add comment:
max_partitions_contributed: maximum number of partition_keys which one privacy_id can have records.

Reply:
Done